server/views/pages/explore.njk

```diff
@@ -23,9 +23,72 @@
     </div>
   </div>
 
+  {# Theme #}
   <div class="row">
     <div class="container">
       <div class="grid">
+        <div class="{{ {s:4, m:8, l:12, xl: 12} | gridClasses }}">
+          <h2 class="heading--large no-v-margin">A drop in the ocean</h2>
```

Comment:
Formalising it makes it acceptable.
Let's stick to public humiliation.
![Shame](https://media.giphy.com/media/3otOKTjlqkF3C1XhFm/giphy.gif)